### DISCUSSION TITLE
A star after the name of most of my notes

### DISCUSSION THREAD

**Discussion by davidszerman** [TriliumNext/Trilium#3208] [Q&A] [answered]
Hello,

Maybe I missed something but my notes have a star after their name (*) ; do you know what it means ?

Thanks

**Comment by sigaloid** [comment, marked as answer]
This means it's cloned elsewhere in the tree:

> Titles of cloned notes in the tree view have an asterisk to the right to easily see that the note is also placed into some other location.

https://github.com/zadam/trilium/wiki/Cloning-notes

**Reply by davidszerman** [reply]
Thanks but when I look at "Note paths", there is only one line... Strange... ?

**Reply by sigaloid** [reply]
Hm, that is odd, perhaps file an issue since it seems to be a bug.
